src/main/java/org/opentripplanner/routing/api/request/StreetMode.java

```diff
@@ -1,140 +1,150 @@
 package org.opentripplanner.routing.api.request;
 
+import java.util.EnumSet;
+import java.util.Set;
 import org.opentripplanner.framework.doc.DocumentedEnum;
 
 public enum StreetMode implements DocumentedEnum<StreetMode> {
   /**
    * No street mode is set. This option is used if we do not want street routing at all in this part
    * of the search.
    */
-  NOT_SET(true, true, true, false, false, false, false, false, false, false),
+  NOT_SET(Feature.ACCESS, Feature.TRANSFER, Feature.EGRESS),
   /**
    * Walk only
    */
-  WALK(true, true, true, true, false, false, false, false, false, false),
+  WALK(Feature.ACCESS, Feature.TRANSFER, Feature.EGRESS, Feature.WALKING),
   /**
    * Bike only
    */
-  BIKE(true, true, true, false, true, false, false, false, false, false),
+  BIKE(Feature.ACCESS, Feature.TRANSFER, Feature.EGRESS, Feature.CYCLING),
   /**
    * Bike to a bike parking area, then walk the rest of the way.
    * <p>
    * Direct mode and access mode only.
    */
-  BIKE_TO_PARK(true, false, false, true, true, false, false, false, true, false),
+  BIKE_TO_PARK(Feature.ACCESS, Feature.WALKING, Feature.CYCLING, Feature.PARKING),
   /**
    * Walk to a bike rental point, bike to a bike rental drop-off point, and walk the rest of the
    * way. This can include bike rental at fixed locations or free-floating services.
    */
-  BIKE_RENTAL(true, true, true, true, true, false, false, true, false, false),
+  BIKE_RENTAL(
```

Comment:
This looks so much nicer!

Can you do me an aesthetic favour and statically import all the enum vallues so `Feature` is not repeated over and over?

Reply:
It seems to be possible if you do it like that: https://github.com/leonardehrenfried/OpenTripPlanner/commit/0404fcd61b442b6bb4231a0c32ab68480602bbf2

I will leave the decision up to you if that is better or not and I will approve without it.

Reply:
@t2gran can decide if we should do it or not.